docs/controllers/services/annotations.md

```diff
@@ -0,0 +1,38 @@
+# Service Annotations
+
+DigitalOcean cloud controller manager watches for Services of type `LoadBalancer` and will create corresponding cloud load balancers matching the Kubernetes service. The cloud loadbalancer can be configured by applying annotations to the Service resource. The following annotations can be used:
+
+### service.beta.kubernetes.io/do-loadbalancer-protocol
+
+The default protocol for DO load balancers. Ports specified in the annotation `service.beta.kubernetes.io/do-loadbalancer-tls-ports` will be overwritten to https. Options are `tcp`, `http` and `https`. Defaults to `tcp`.
+
+### service.beta.kubernetes.io/do-loadbalancer-tls-ports 
+
+Specify which ports of the loadbalancer should use the https protocol. This is a comma separated list of ports (e.g. 443,6443,7443).
+
+### service.beta.kubernetes.io/do-loadbalancer-tls-passthrough
+
+Specify whether the DigitalOcean loadbalancer should pass encrypted data to backend droplets. This is optional. Options are `true` or `false`. Defaults to `false`.
```

Comment:
Nit: we're talking sometimes about `DO load balancers`, here it is `DigitalOcean loadbalancer`. It seems like (according to the https://www.digitalocean.com/products/load-balancer/ page) it should be called:

```
* DigitalOcean Load Balancer
* DigitalOcean Load Balancers
```

So I suggest we always use this name whenever we want to refer to the load balancer product.